cassandra/src/main/resources/reference.conf

```diff
@@ -0,0 +1,5 @@
+cassandra.roots: "127.0.0.1"
```

Comment:
you can specify at such
cassandra{
  keyspace: "test"
  keyname: "testkey"
  columnfamily: "testfamily"
  columnname: "testcolumn"
}

Reply:
Changed